apisix/plugins/chaitin-waf.lua

```diff
@@ -48,8 +52,11 @@ local match_schema = {
 local plugin_schema = {
     type = "object",
     properties = {
-        -- TODO: we should add a configuration "mode" here
-        -- It can be one of off, block and monitor
+        mode = {
+            type = "string",
+            enum = { "off", "monitor", "block", nil },
+            default = nil,
```

Comment:
We don't need to explicitly specify nil in enum and default value. When this field is not declared as required in the JSON schema, it can be nil, and the default value is also nil.

Reply:
This does not affect the use of this feature, which is merged first and then an issue is created to track it.